src/main/scala/beam/analysis/RideHailUtilizationCollector.scala

```diff
@@ -71,11 +79,23 @@ class RideHailUtilizationCollector(beamSvc: BeamServices)
       case pte: PathTraversalEvent if pte.vehicleId.toString.contains("rideHailVehicle-") =>
         handle(pte)
       case mc: ModeChoiceEvent =>
-        if (mc.mode == "ride_hail")
+        if (mc.mode == BeamMode.RIDE_HAIL.value)
           rideHailChoices += 1
-        if (mc.availableAlternatives.contains("RIDE_HAIL"))
+        else if (mc.mode == BeamMode.RIDE_HAIL_POOLED.value)
+          rideHailPooledChoices += 1
+        if (mc.availableAlternatives.contains("RIDE_HAIL:"))
```

Comment:
This is a slight deviation from what it was, should the colon be there?

Reply:
Yes, it is. Here is how the string is created: https://github.com/LBNL-UCB-STI/beam/blob/265becd59ff168ecd43ae6686fc8332dfadd95fd/src/main/scala/beam/agentsim/agents/modalbehaviors/ChoosesMode.scala#L957

Reply:
But if there is only 1 item in the collection then there will be no colon, leaving a single to fall through the cracks

Reply:
@JustinPihony fixed